### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by KostinP**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/375

### CODE REVIEW

File: src/FileService.Business/Commands/Image/AddNewImageCommand.cs [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+using LT.DigitalOffice.FileService.Validation.Interfaces;
 using LT.DigitalOffice.Kernel.Extensions;
 using LT.DigitalOffice.Kernel.FluentValidationExtensions;
 using Microsoft.AspNetCore.Http;
 using System;
 
-namespace LT.DigitalOffice.FileService.Business
+namespace LT.DigitalOffice.FileService.Business.Commands.Image
 {
     public class AddNewImageCommand : IAddNewImageCommand
```
Comment (MariyaSin) [inline src/FileService.Business/Commands/Image/AddNewImageCommand.cs:14]: I think if you write "add" it means "new", it unnecessary to use "new". I think will be better use "AddImageCommand"

File: test/FileService.Business.UnitTests/Commands/File/AddNewFileCommandTests.cs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+using LT.DigitalOffice.FileService.Validation.Interfaces;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.FileService.Business.UnitTests.Commands.File
+{
+    public class AddNewFileCommandTests
+    {
```
Comment (MariyaSin) [inline test/FileService.Business.UnitTests/Commands/File/AddNewFileCommandTests.cs:18]: Please, rewrite all Tests names. For example, you rewrite the command name to AddFileCommand, but do not rewrite Test name to AddFileCommandTests

File: src/FileService.Business/Commands/File/GetFileByIdCommand.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using LT.DigitalOffice.FileService.Business.Commands.File.Interfaces;
+using LT.DigitalOffice.FileService.Data.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.FileService.Models.Dto.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.FileService.Business.Commands.File
+{
+    public class GetFileByIdCommand : IGetFileByIdCommand
```
[the remark this answers is not in the document]
Comment (KostinP) [reply src/FileService.Business/Commands/File/GetFileByIdCommand.cs]: Fixed

File: src/FileService.Business/Commands/Image/AddNewImageCommand.cs (hunk 1 above)
Comment (KostinP) [reply src/FileService.Business/Commands/Image/AddNewImageCommand.cs]: Fixed

File: test/FileService.Business.UnitTests/Commands/File/AddNewFileCommandTests.cs (hunk 2 above)
Comment (KostinP) [reply test/FileService.Business.UnitTests/Commands/File/AddNewFileCommandTests.cs]: Fixed

File: test/FileService.Data.UnitTests/AddNewFileRepositoryTests.cs [hunk 4]
```diff
@@ -9,10 +9,10 @@ namespace LT.DigitalOffice.FileService.Data.UnitTests
 {
     public class AddNewFileRepositoryTests
     {
```
Comment (MariyaSin) [inline test/FileService.Data.UnitTests/AddNewFileRepositoryTests.cs:11]: I think will be better AddFileRepositoryTests without "New"

File: test/FileService.Data.UnitTests/DisableFileByIdRepositoryTests.cs [hunk 5]
```diff
@@ -10,55 +10,55 @@ namespace LT.DigitalOffice.FileService.Data.UnitTests
 {
     class DisableFileByIdRepositoryTests
     {
```
Comment (MariyaSin) [inline test/FileService.Data.UnitTests/DisableFileByIdRepositoryTests.cs:12]: Maybe without "ById"?

File: test/FileService.Data.UnitTests/GetFileByIdRepositoryTests.cs [hunk 6]
```diff
@@ -11,21 +11,21 @@ namespace LT.DigitalOffice.FileService.Data.UnitTests
 {
     public class GetFileByIdRepositoryTests
     {
```
Comment (MariyaSin) [inline test/FileService.Data.UnitTests/GetFileByIdRepositoryTests.cs:13]: Maybe without "ById"?

File: test/FileService.Validation.UnitTests/AddNewFileValidatorTests.cs [hunk 7]
```diff
@@ -1,77 +1,77 @@
...
-using FluentValidation.TestHelper;
+﻿using FluentValidation.TestHelper;
 using LT.DigitalOffice.FileService.Models.Dto.Models;
+using LT.DigitalOffice.FileService.Validation.Interfaces;
 using NUnit.Framework;
 
 namespace LT.DigitalOffice.FileService.Validation.UnitTests
 {
     public class AddNewFileRequestValidatorTests
     {
```
Comment (MariyaSin) [inline test/FileService.Validation.UnitTests/AddNewFileValidatorTests.cs:9]: Maybe without "New"?

File: test/FileService.Validation.UnitTests/AddNewImageValidatorTests.cs [hunk 8]
```diff
@@ -1,64 +1,64 @@
...
-using FluentValidation.TestHelper;
+﻿using FluentValidation.TestHelper;
 using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.FileService.Validation.Interfaces;
 using NUnit.Framework;
 
 namespace LT.DigitalOffice.FileService.Validation.UnitTests
 {
     public class AddNewImageRequestValidatorTests
     {
```
Comment (MariyaSin) [inline test/FileService.Validation.UnitTests/AddNewImageValidatorTests.cs:9]: Maybe without "New"?

File: test/FileService.Validation.UnitTests/FileRequestValidatorTests.cs [hunk 9]
```diff
@@ -1,77 +1,78 @@
...
+﻿using FluentValidation.TestHelper;
 using LT.DigitalOffice.FileService.Models.Dto.Models;
+using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.FileService.Validation.Interfaces;
 using NUnit.Framework;
 
 namespace LT.DigitalOffice.FileService.Validation.UnitTests
 {
     public class AddNewFileRequestValidatorTests
     {
```
Comment (MariyaSin) [inline test/FileService.Validation.UnitTests/FileRequestValidatorTests.cs:10]: File name and class name have a difference

File: test/FileService.Validation.UnitTests/ImageRequestValidatorTests.cs [hunk 10]
```diff
@@ -0,0 +1,64 @@
+﻿using FluentValidation.TestHelper;
+using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.FileService.Validation.Interfaces;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.FileService.Validation.UnitTests
+{
+    public class AddNewImageRequestValidatorTests
```
Comment (MariyaSin) [inline test/FileService.Validation.UnitTests/ImageRequestValidatorTests.cs:8]: Class name and file name have a difference

File: test/FileService.Mappers.UnitTests/Db/FileMapperTests.cs [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+using LT.DigitalOffice.FileService.Models.Db;
+using LT.DigitalOffice.FileService.Models.Dto.Models;
+using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.UnitTestKernel;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.FileService.Mappers.UnitTests.Db
+{
+    public class DbFileMapperTests
```
Comment (MariyaSin) [inline test/FileService.Mappers.UnitTests/Db/FileMapperTests.cs:14]: Class name and file name have a difference

File: test/FileService.Data.UnitTests/AddNewImageRepositoryTests.cs [hunk 12]
```diff
@@ -9,10 +9,10 @@ namespace LT.DigitalOffice.FileService.Data.UnitTests
 {
     public class AddNewImageRepositoryTests
```
[the remark this answers is not in the document]
Comment (KostinP) [reply test/FileService.Data.UnitTests/AddNewImageRepositoryTests.cs]: Fixed

File: test/FileService.Data.UnitTests/AddNewFileRepositoryTests.cs (hunk 4 above)
Comment (KostinP) [reply test/FileService.Data.UnitTests/AddNewFileRepositoryTests.cs]: Fixed

File: test/FileService.Data.UnitTests/DisableFileByIdRepositoryTests.cs (hunk 5 above)
Comment (KostinP) [reply test/FileService.Data.UnitTests/DisableFileByIdRepositoryTests.cs]: Fixed

File: test/FileService.Data.UnitTests/GetFileByIdRepositoryTests.cs (hunk 6 above)
Comment (KostinP) [reply test/FileService.Data.UnitTests/GetFileByIdRepositoryTests.cs]: Fixed

File: test/FileService.Validation.UnitTests/AddNewFileValidatorTests.cs (hunk 7 above)
Comment (KostinP) [reply test/FileService.Validation.UnitTests/AddNewFileValidatorTests.cs]: Fixed

File: test/FileService.Validation.UnitTests/AddNewImageValidatorTests.cs (hunk 8 above)
Comment (KostinP) [reply test/FileService.Validation.UnitTests/AddNewImageValidatorTests.cs]: Fixed

File: test/FileService.Validation.UnitTests/FileRequestValidatorTests.cs (hunk 9 above)
Comment (KostinP) [reply test/FileService.Validation.UnitTests/FileRequestValidatorTests.cs:10]: Fixed

File: test/FileService.Validation.UnitTests/ImageRequestValidatorTests.cs (hunk 10 above)
Comment (KostinP) [reply test/FileService.Validation.UnitTests/ImageRequestValidatorTests.cs]: Fixed

File: test/FileService.Mappers.UnitTests/Db/FileMapperTests.cs (hunk 11 above)
Comment (KostinP) [reply test/FileService.Mappers.UnitTests/Db/FileMapperTests.cs]: Fixed

**Review by MariyaSin — APPROVED** [review]

File: Common.Configurations.targets [hunk 13]
```diff
@@ -37,8 +37,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests'))">
-    <PackageReference Include="LT.DigitalOffice.UnitTestKernel" Version="1.0.2" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
+    <PackageReference Include="LT.DigitalOffice.UnitTestKernel" Version="1.0.4" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
```
Comment (BelehovEgor) [inline Common.Configurations.targets:41]: We have it in UnitTestKernel, delete this

File: src/FileService/Controllers/FileController.cs [hunk 14]
```diff
@@ -9,23 +10,23 @@ namespace LT.DigitalOffice.FileService.Controllers
     [Route("[controller]")]
     public class FileController : ControllerBase
     {
-        [HttpPost("addNewFile")]
+        [HttpPost("add")]
         public Guid AddNewFile(
```
Comment (BelehovEgor) [inline src/FileService/Controllers/FileController.cs:14]: can rename to Add

File: src/FileService/Controllers/FileController.cs [hunk 15]
```diff
@@ -9,23 +10,23 @@ namespace LT.DigitalOffice.FileService.Controllers
...
+            [FromBody] FileRequest request,
+            [FromServices] IAddFileCommand command)
         {
             return command.Execute(request);
         }
 
-        [HttpGet("getFileById")]
-        public File GetFileById([FromServices] IGetFileByIdCommand command, [FromQuery] Guid fileId)
+        [HttpGet("get")]
+        public FileInfo GetFile([FromServices] IGetFileCommand command, [FromQuery] Guid fileId)
```
Comment (BelehovEgor) [inline src/FileService/Controllers/FileController.cs:22]: rename to Get

File: src/FileService/Controllers/FileController.cs [hunk 16]
```diff
@@ -9,23 +10,23 @@ namespace LT.DigitalOffice.FileService.Controllers
...
         {
             return command.Execute(fileId);
         }
 
-        [HttpDelete("disableFileById")]
-        public void DisableFileById(
-            [FromServices] IDisableFileByIdCommand command,
+        [HttpDelete("disable")]
+        public void DisableFile(
```
Comment (BelehovEgor) [inline src/FileService/Controllers/FileController.cs:28]: rename to disable

File: src/FileService/Controllers/ImageController.cs [hunk 17]
```diff
@@ -9,10 +9,10 @@ namespace LT.DigitalOffice.FileService.Controllers
     [Route("[controller]")]
     public class ImageController : ControllerBase
     {
-        [HttpPost("addNewImage")]
+        [HttpPost("add")]
         public Guid AddNewImage(
```
Comment (BelehovEgor) [inline src/FileService/Controllers/ImageController.cs:13]: rename to Add

**Review by BelehovEgor — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

**Review by djino97 — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Pull request merged**